.circleci/config.yml

```diff
@@ -132,27 +132,31 @@ jobs:
             LAST_COMMIT=`git log -1 --pretty=%B`
```

Comment:
move the `if [ ! -z $CIRCLE_PULL_REQUEST ]; then` block here,
so the `else` block can also share the logic?